flink-yarn-tests/src/test/java/org/apache/flink/yarn/YARNSessionFIFOITCase.java

```diff
@@ -272,6 +275,7 @@ public void testJavaAPI() throws Exception {
 		LOG.info("Shutting down cluster. All tests passed");
 		// shutdown cluster
 		yarnCluster.shutdown();
+		clusterDescriptor.close();
```

Comment:
Closing in finally or try-with-resource not needed?

Reply:
will refactor it